driver/src/test/scala/org/mongodb/scala/internal/ScalaObservableSpec.scala

```diff
@@ -0,0 +1,296 @@
+/*
+ * Copyright 2015 MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.mongodb.scala.internal
+
+import scala.util.{ Failure, Success }
+
+import com.mongodb.MongoException
+
+import org.mongodb.scala._
+import org.scalatest.{ FlatSpec, Matchers }
+
+class ScalaObservableSpec extends FlatSpec with Matchers {
```

Comment:
Even though there is good test coverage, are these tests asserting everything they need to be? They are all asserting only about the last value seen.